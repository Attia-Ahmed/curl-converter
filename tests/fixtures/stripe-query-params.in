curl https://api.stripe.com/v1/customers/search -u sk_test_sabcdef0987654321: --data-urlencode query="name:'fakename' AND metadata['foo']:'bar'" -G
