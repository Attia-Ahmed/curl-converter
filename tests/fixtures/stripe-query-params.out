Http::withBasicAuth('sk_test_sabcdef0987654321', '')
    ->get('https://api.stripe.com/v1/customers/search', [
        'query' => 'name:\'fakename\' AND metadata[\'foo\']:\'bar\'',
    ]);
